com.unity.ml-agents.extensions/Tests/Editor/Sensors/PoseExtractorTests.cs

```diff
@@ -0,0 +1,122 @@
+using UnityEngine;
+using NUnit.Framework;
+using Unity.MLAgents.Extensions.Sensors;
+
+namespace Unity.MLAgents.Extensions.Tests.Sensors
+{
+    public class HierarchyUtilTests
```

Comment:
nit: rename test to match new class name

Reply:
Updated the tests (and renamed a few local variables).

```
 $ git grep -i hierarchyutil | wc -l
       0
```